python/paddle/text/datasets/wmt14.py

```diff
@@ -95,9 +101,21 @@ class WMT14(Dataset):
 
     """
 
+    mode: _Wmt14DataSetMode
+    data_file: str | None
+    dict_size: int
+    out_dict: dict[str, int]
+    src_ids: list[npt.NDArray[np.int_]]
+    trg_ids: list[npt.NDArray[np.int_]]
+    trg_ids_next: list[npt.NDArray[np.int_]]
```

Comment:
```suggestion
    src_ids: list[list[int]]
    trg_ids: list[list[int]]
    trg_ids_next: list[list[int]]
```

实例中的属性貌似应该是 `list[list[int]]` ，是在 `__getitem__` 中转换为 `npt.NDArray` 的 ～

另外，貌似少了两个属性

``` python
src_dict: dict[str, int]
trg_dict: dict[str, int]
```

Reply:
收到

Reply:
收到